sql/insert.go

```diff
@@ -88,6 +95,22 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 				b.Put(key, string(t))
```

Comment:
Non-unique secondary keys are encoded as `<secondary-key>/<primary-key> -> NULL`. Perhaps unique secondary keys should be encoded as `<secondary-key> -> <primary-key>`. This makes unique for non-unique indexes differ, but allows the continued use of `CPut` which is a pretty big advantage.